lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2016/data/ashrae_90_1_2016.prototype_inputs.json

```diff
@@ -1246,9 +1246,9 @@
       "pszac_oa_damper_schedule_2": null,
       "pszac_operation_schedule_3": null,
       "pszac_oa_damper_schedule_3": null,
-      "occ_sensing_exterior_lighting_power": 7402.3,
+      "occ_sensing_exterior_lighting_power": 4717.264,
       "occ_sensing_exterior_lighting_schedule": "OfficeMedium Exterior_lighting_schedule_b",
-      "nondimming_exterior_lighting_power": 519.2,
+      "nondimming_exterior_lighting_power": 519.1875,
```

Comment:
Are these small differences intended? If not we might want to either make the same adjustment to earlier version of 90.1 or use the same value as the earlier version of 90.1.

Reply:
Yes. These are intentional. I use the exact numbers from our worksheet to update the google spreadsheet. I am fine either way (reverting these small changes or updating previous code versions' numbers)

Reply:
I see, thanks! Perhaps updating previous code version is better?